requirements.txt

```diff
@@ -6,7 +6,8 @@ torchvision==0.8.2
 transformers==4.2.2
 pytorch-tabnet==3.1.1
 datasets==1.2.1
-pandas==1.1.2
+pandas==1.1.5 ; python_version < "3.7"
```

Comment:
Is 1.2.2 not supported for 3.6?

Reply:
Nope, failed for CI testing and they mentioned 3.7 and upwards for pandas. https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html